qiita_db/metadata_template/base_metadata_template.py

```diff
@@ -705,9 +729,23 @@ def delete_column(self, column_name):
                     column_name)
 
         with qdb.sql_connection.TRN:
-            sql = 'ALTER TABLE qiita.%s%d DROP COLUMN %s' % (
-                self._table_prefix, self._id, column_name)
-            qdb.sql_connection.TRN.add(sql)
+            table_name = 'qiita.{0}{1}'.format(self._table_prefix, self._id)
+            # deleting from all samples
+            sql = """UPDATE {0}
```

Comment:
If I read this right, we might want to reiterate the '-' operator is a JSONB operation?